src/Gir/Marshal/Enumeration.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace Gir
+{
+	public partial class Enumeration
+	{
+		public string DefaultValue => "default(" + Name + ")";
```

Comment:
Is this safe? default(enum) = (E)0. But what happens if there is no enum type with the value of 0?

Reply:
Yeah, I added that as a placeholder, will add TODO:

It should probably use the first element.